src/DefinitionResolver.php

```diff
@@ -174,7 +174,11 @@ public function createDefinitionFromNode(Node $node, string $fqn = null): Defini
         $def->fqn = $fqn;
 
         // Determines whether the suggestion will show after "new"
-        $def->canBeInstantiated = $node instanceof Node\Statement\ClassDeclaration;
+        $def->canBeInstantiated = (
+            $node instanceof Node\Statement\ClassDeclaration &&
+            // check whether it is not an abstract class
+            ($node->abstractOrFinalModifier === null || $node->abstractOrFinalModifier->kind !== PhpParser\TokenKind::AbstractKeyword)
```

Comment:
In which case `$node->abstractOrFinalModifier->kind !== PhpParser\TokenKind::AbstractKeyword` is `true`